packages/react-reconciler/src/ReactFiberReconciler.js

```diff
@@ -93,8 +93,8 @@ export type HostConfig<T, P, I, TI, HI, PI, C, CC, CX, PL> = {
   ): number,
   cancelDeferredCallback(callbackID: number): void,
 
-  prepareForCommit(): void,
-  resetAfterCommit(): void,
+  prepareForCommit(containerInfo: any): void,
```

Comment:
These should have the type `C`. The return `data` type needs some new generic type argument.

Reply:
Updated the typing (commit data referenced as `CD`), it seems to pass the flow test. I'd have expected to have to add an `any` into a generic type instantiation somewhere, but Flow seems able to infer it?